src/sentry/workflow_engine/processors/action.py

```diff
@@ -73,11 +76,17 @@ def update_workflow_fire_histories(
         ).values_list("workflow_id", flat=True)
     )
 
+    updates: dict[str, bool] = {}
```

Comment:
could we just pass in this typed dict instead of the two different attributes? then we could just update the type if we want to add / remove any attributes.

Reply:
okie